index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <section class="top-nav">
+            <button id="header-btn1">
+                <a href="">
+                    <img src="./icons/logo.png" alt="">
+                </a>
+            </button>
+
+            <button id="header-btn2">
+                <nav href="">
+                    <img src="./icons/Union.png" alt="">
+                </nav>
+            </button>
+        </section>
+
+        <div class="main-background">
+            <img id="icon-1" src="./icons/Ellipse 11.png" alt="">
+            <img id="icon-2" src="./icons/Rectangle 7.png" alt="">
+            <img id="icon-3" src="./icons/Ellipse 5.png" alt="">
+            <img id="icon-4" src="./icons/Ellipse 3.png" alt="">
+            <img id="icon-5" src="./icons/Ellipse 12.png" alt="">
+            <img id="icon-6" src="./icons/Ellipse 4.png" alt="">
+            <img id="icon-7" src="./icons/Ellipse 8.png" alt="">
+            <img id="icon-8" src="./icons/Ellipse 13.png" alt="">
+            <img id="icon-9" src="./icons/Ellipse 14.png" alt="">
+            <img id="icon-10" src="./icons/Ellipse 15.png" alt="">
+            <img id="icon-11" src="./icons/Rectangle 4.png" alt="">
+            <img id="icon-12" src="./icons/Rectangle 5.png" alt="">
+            <img id="icon-13" src="./icons/Rectangle 6.png" alt="">
+        </div>
+
+        <section class="headline">
+            <div class="main-text1">
+              <h2>I’m Mesak Glad to see you!</h2>
+            </div>
+
+            <div class="main-text2">
+                <h3>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </h3>
+            </div>
+
+            <div class="socials-text">
+                <div class="supporting-text">
+                    <h4>LET’S CONNECT</h4>
+                </div>
+            </div>
```

Comment:
- [ ]  Hello @MesakDuduCoder , Good job so far😃,  As I can see, your project looks fine, but you need to fix your `margin` so that your website will look exactly like the design (e.g. font, colors, images, text, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.)👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![figma-mask](https://user-images.githubusercontent.com/57408419/220685897-cbe31aa0-bfe8-484c-b6d1-156731d26f82.jpg)| |![mask](https://user-images.githubusercontent.com/57408419/220686592-fc1a3dbf-52c1-4eb4-b81f-cfb90e0148f4.jpg) | |